helper/forwarding/util.go

```diff
@@ -178,6 +178,7 @@ func NewRPCResponseWriter() *RPCResponseWriter {
 		statusCode: 200,
 	}
 	//w.header.Set("Content-Type", "application/octet-stream")
+	w.Header().Set("Cache-Control", "no-store")
```

Comment:
Is this necessary? The RPC code (see vault/request_forwarding.go line 375-ish) copies headers received from the response from the forwarded server over to the reply. I'd expect the header to already be in there.